models/pull.go

```diff
@@ -26,8 +26,8 @@ var PullRequestQueue = sync.NewUniqueQueue(setting.Repository.PullRequestQueueLe
 type PullRequestType int
 
 const (
-	PULL_REQUEST_GOGS PullRequestType = iota
-	PLLL_ERQUEST_GIT
+	PULL_REQUEST_GITEA PullRequestType = iota
```

Comment:
Constants should be named lowerCamelCase for unexported ones and UpperCamelCase for exported ones.
https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps